src/api/routers/participants/participantsKeyPairs.spec.ts

```diff
@@ -81,10 +112,11 @@ describe('#getParticipantKeyPairs', () => {
   });
 
   test.each([
-    [10, []],
-    [11, oneKeyPair],
-    [12, multipleKeyPairs],
```

Comment:
This test was previously failing because it was expecting to return all keypairs, not just the enabled ones. I've added a test for the disabled keypair case and extracted a bunch of magic numbers to constants.